docs/articles/documentation/test-api/README.md

```diff
@@ -22,4 +22,5 @@ It contains the following subsections
 * [Pausing the Test](pausing-the-test.md)
 * [Handling Native Dialogs](handling-native-dialogs.md)
 * [Working with \<iframes\>](working-with-iframes.md)
-* [Debugging](debugging.md)
\ No newline at end of file
+* [Debugging](debugging.md)
+* [TypeScript Support](typescript-support.md)
```

Comment:
Imagine I never worked with TestCafe before and heard it supports TypeScript, where would I look for the documentation for that? Somewhere in the very end of the content table? Guys, it's **BASIC** stuff it should be my **FIRST** steps in learning TestCafe if I prefer to use TypeScript.